addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1073,6 +1116,43 @@ def create_group(self, partners_to, default_display_mode=False, name=''):
         channel._broadcast(channel.channel_member_ids.partner_id.ids)
         return channel
 
+    def _create_sub_channel(self, from_message_id=None, name=None):
+        self.ensure_one()
+        message = self.env["mail.message"]
+        if from_message_id:
+            message = self.env["mail.message"].search([("id", "=", from_message_id)])
+        sub_channel = self.create(
+            {
+                "channel_member_ids": [Command.create({"partner_id": self.env.user.partner_id.id})],
+                "channel_type": "channel",
+                "from_message_id": message.id,
+                "name": name or (message.body.striptags()[:30] if message else _("New Thread")),
+                "parent_channel_id": self.id,
+            }
+        )
+        self.env.user.partner_id._bus_send_store(
+            Store(sub_channel).add(sub_channel, {"forceOpen": True})
```

Comment:
Isn't `forceOpen` going to force it for all tabs? I don't think it is something we should send on the bus.

Only RPC result should have `forceOpen` for current tab.